docs/update-version.sh

```diff
@@ -9,5 +9,5 @@ then
 fi
 
 sed --in-place --regexp-extended --expression \
-  's/'"${DAGGER_VERSION_FROM:-0.2.4}"'/'"${DAGGER_VERSION_TO:-0.2.7}"'/g' \
+  's/'"${DAGGER_VERSION_FROM:-0.2.7}"'/'"${DAGGER_VERSION_TO:-0.2.8}"'/g' \
```

Comment:
We should probably escape these version dots, otherwise they'll match any char instead of an explicit `.`.

Reply:
:+1: rushed PR. I'll fix it. Thx Helder